frontend/config/routes.rb

```diff
@@ -9,6 +9,7 @@
   patch '/checkout/update/:state', to: 'checkout#update', as: :update_checkout
   get '/checkout/:state', to: 'checkout#edit', as: :checkout_state
   get '/checkout', to: 'checkout#edit', as: :checkout
+  post '/checkout/remove_credit_card/:credit_card', to: 'checkout#remove_credit_card', as: :remove_credit_card
```

Comment:
@aplegatt let's move that to API controller please - https://github.com/spree/spree/blob/master/api/app/controllers/spree/api/v2/storefront/account/credit_cards_controller.rb - under `destroy` action